person.rb

```diff
@@ -30,10 +31,10 @@ def correct_name
   end
 end
```

Comment:
- [ ] The same here, A person has many rentals and A rental belongs to a person. Let's create A method for adding rentals !!